osquery/tables/networking/windows/interfaces.cpp

```diff
@@ -67,35 +67,28 @@ void genInterfaceDetail(const IP_ADAPTER_ADDRESSES* adapter, Row& r) {
     auto& results = req1.results();
     if (!results.empty()) {
       std::string sPlaceHolder;
-      unsigned long long ullPlaceHolder = 0;
+      using ull = unsigned long long;
+      const auto defaultValue = ull{0};
```

Comment:
Is there any reason for storing casted 0 and saving several characters by `using ull`?
To my understanding, it just obfuscates the code when the developer reading that simple line below, has to check two additional references to the ull and defaultValue, when those could simply be avoided.

Reply:
Make sense, have removed it.

Thank you for review, @guliashvili !